### ISSUE TITLE
[bug]: Install model failed with modal manager

### ISSUE THREAD

**Issue by xyy94813** [invoke-ai/InvokeAI#6040]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Linux

### GPU vendor

Nvidia (CUDA)

### GPU model

Nvidia 1060

### GPU VRAM

6GB

### Version number

v4.0.0rv5

### Browser

chrome ....

### What happened

Install model `CyberRealistic_Negative_v3.pt` by model manager.
Download completed but install failed.

```shell
2024-03-24 01:52:41 [2024-03-23 17:52:41,224]::[ModelInstallService]::INFO --> Model download complete: https://huggingface.co/cyberdelia/CyberRealistic/resolve/main/CyberRealistic_V4.1_FP16.safetensors
2024-03-24 01:52:41 [2024-03-23 17:52:41,227]::[ModelInstallService]::INFO --> Model install started: https://huggingface.co/cyberdelia/CyberRealistic/resolve/main/CyberRealistic_V4.1_FP16.safetensors
2024-03-24 01:52:41 Exception in thread Thread-1 (_install_next_item):
2024-03-24 01:52:41 Traceback (most recent call last):
2024-03-24 01:52:41   File "/usr/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
2024-03-24 01:52:41     self.run()
2024-03-24 01:52:41   File "/usr/lib/python3.11/threading.py", line 975, in run
2024-03-24 01:52:41     self._target(*self._args, **self._kwargs)
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 458, in _install_next_item
2024-03-24 01:52:41     self._register_or_install(job)
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 488, in _register_or_install
2024-03-24 01:52:41     key = self.install_path(job.local_path, job.config_in)
2024-03-24 01:52:41           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 186, in install_path
2024-03-24 01:52:41     info: AnyModelConfig = ModelProbe.probe(Path(model_path), config, hash_algo=self._app_config.hashing_algorithm)
2024-03-24 01:52:41                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/backend/model_manager/probe.py", line 147, in probe
2024-03-24 01:52:41     probe = probe_class(model_path)
2024-03-24 01:52:41             ^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/backend/model_manager/probe.py", line 389, in __init__
2024-03-24 01:52:41     self.checkpoint = ModelProbe._scan_and_load_checkpoint(model_path)
2024-03-24 01:52:41                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41   File "/opt/invokeai/invokeai/backend/model_manager/probe.py", line 330, in _scan_and_load_checkpoint
2024-03-24 01:52:41     return safetensors.torch.load_file(model_path)
2024-03-24 01:52:41            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41   File "/opt/venv/invokeai/lib/python3.11/site-packages/safetensors/torch.py", line 308, in load_file
2024-03-24 01:52:41     with safe_open(filename, framework="pt", device=device) as f:
2024-03-24 01:52:41          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 01:52:41 safetensors_rust.SafetensorError: Error while deserializing header: MetadataIncompleteBuffer
```

Got similar error when install `InvokeAI/ip_adapter_sdxl`

```
2024-03-24 00:37:08 [2024-03-23 16:37:08,453]::[ModelInstallService]::INFO --> Model install started: InvokeAI/ip_adapter_sdxl
2024-03-24 00:37:08 Exception in thread Thread-1 (_install_next_item):
2024-03-24 00:37:08 Traceback (most recent call last):
2024-03-24 00:37:08   File "/usr/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
2024-03-24 00:37:08     self.run()
2024-03-24 00:37:08   File "/usr/lib/python3.11/threading.py", line 975, in run
2024-03-24 00:37:08     self._target(*self._args, **self._kwargs)
2024-03-24 00:37:08   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 458, in _install_next_item
2024-03-24 00:37:08     self._register_or_install(job)
2024-03-24 00:37:08   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 488, in _register_or_install
2024-03-24 00:37:08     key = self.install_path(job.local_path, job.config_in)
2024-03-24 00:37:08           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08   File "/opt/invokeai/invokeai/app/services/model_install/model_install_default.py", line 186, in install_path
2024-03-24 00:37:08     info: AnyModelConfig = ModelProbe.probe(Path(model_path), config, hash_algo=self._app_config.hashing_algorithm)
2024-03-24 00:37:08                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08   File "/opt/invokeai/invokeai/backend/model_manager/probe.py", line 154, in probe
2024-03-24 00:37:08     fields["base"] = fields.get("base") or probe.get_base_type()
2024-03-24 00:37:08                                            ^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08   File "/opt/invokeai/invokeai/backend/model_manager/probe.py", line 724, in get_base_type
2024-03-24 00:37:08     state_dict = torch.load(model_file, map_location="cpu")
2024-03-24 00:37:08                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08   File "/opt/venv/invokeai/lib/python3.11/site-packages/torch/serialization.py", line 1005, in load
2024-03-24 00:37:08     with _open_zipfile_reader(opened_file) as opened_zipfile:
2024-03-24 00:37:08          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08   File "/opt/venv/invokeai/lib/python3.11/site-packages/torch/serialization.py", line 457, in __init__
2024-03-24 00:37:08     super().__init__(torch._C.PyTorchFileReader(name_or_buffer))
2024-03-24 00:37:08                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-24 00:37:08 RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory
```

### What you expected to happen

Download success

### Additional context

WIN 10/Docker/Intel

**Comment by psychedelicious** [comment]
I cannot reproduce this.

These model files are fairly large. I wonder if they fully downloaded? Could be a network issue, is this still an issue if you try again?

Please try downloading the model directly:

```
wget https://huggingface.co/cyberdelia/CyberRealistic/resolve/main/CyberRealistic_V4.1_FP16.safetensors
```

Then use the local path mode to install the model. Does this work?

**Comment by xyy94813** [comment]
I had resolved it by follow step:

1. restart InvokeAI
2. reinstall model.

------

Reinstall model is not working before restart InvokeAI...
I suspect that this is caused by manually removing tasks from the download queue...

**Comment by psychedelicious** [comment]
There's a problem where the app hangs if an install broke in an unexpected way. I think you ran into this issue. It's fixed in #6048
